spring-cloud-commons/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+  "hints": [],
+  "groups": [],
+  "properties": [
+    {
+      "sourceType": "org.springframework.cloud.autoconfigure.RefreshAutoConfiguration",
```

Comment:
I think the `sourceType` can go away as it doesn't really match the contract. This is where the property is _used_ not where the property is _defined_. There is no such place with a `setEnabled` and `getEnabled` methods so we shouldn't have a `sourceType` either.

(This applied to all `sourceType` attributes in this PR).